api/cases/case.js

```diff
@@ -11,10 +12,32 @@ function getCaseEvents(caseId, userId, options, caseType = 'Benefit', jurisdicti
     return generateRequest(`${config.services.ccd_data_api}/caseworkers/${userId}/jurisdictions/${jurisdiction}/case-types/${caseType}/cases/${caseId}/events`, options)
 }
 
-function getCaseWithEvents(caseId, userId, options, caseType = 'Benefit', jurisdiction = 'SSCS') {
+function postHearing(caseId, userId, options, jurisdictionId = 'SSCS') {
+    options.body = {
+        case_id: caseId,
+        jurisdiction: jurisdictionId,
+        panel: [{identity_token: 'string', name: userId}],
+        start_date: (new Date()).toISOString()
+    };
+    
+    return generatePostRequest(`${config.services.coh_cor_api}/continuous-online-hearings`, options)
+        .then(hearing => hearing.online_hearing_id);
+ }
+ 
+function getHearingId(caseId, userId, options) {
```

Comment:
Is this right that if we can't find a COR for a case then we create one?

Reply:
why's that?

Reply:
cause no one has the functionality to create a COH other than us atm
- not sscs
- not ccd